packages/p2p-v2/src/pages/my-profile/screens/PaymentMethods/PaymentMethodsList/PaymentMethodsListContent.tsx

```diff
@@ -0,0 +1,132 @@
+import { Text } from '@deriv-com/ui/dist/components/Text';
+import { ConfirmDeletePaymentMethodModal } from '../../../../../components/Modals';
+import { PaymentMethodCard } from '../../../../../components/PaymentMethodCard';
+import { PAYMENT_METHOD_CATEGORIES } from '../../../../../constants';
+import { TFormState } from '../../../../../reducers/types';
+import AddNewButton from './AddNewButton';
+
+type TPaymentMethodsGroup = Record<
+    string,
+    {
+        paymentMethods: TAdvertiserPaymentMethods;
+        title: string;
+    }
+>;
+
+type TPaymentMethodsListContentProps = {
+    formState: TFormState;
+    isMobile: boolean;
+    onAdd: (selectedPaymentMethod?: TSelectedPaymentMethod) => void;
+    onDelete: (selectedPaymentMethod?: TSelectedPaymentMethod) => void;
+    onEdit: (selectedPaymentMethod?: TSelectedPaymentMethod) => void;
+    onResetFormState: () => void;
+    p2pAdvertiserPaymentMethods: TAdvertiserPaymentMethods;
+};
+
+/**
+ * @component This component is used to display a list of payment methods. It's the content of the PaymentMethodsList component, when the list is not empty
+ * @param formState - The current state of the form
+ * @param isMobile - Whether the current device is mobile or not
+ * @param p2pAdvertiserPaymentMethods - The list of payment methods
+ * @returns {JSX.Element}
+ * @example <PaymentMethodsListContent formState={formState} isMobile={isMobile} p2pAdvertiserPaymentMethods={p2pAdvertiserPaymentMethods} />
+ * **/
+const PaymentMethodsListContent = ({
+    formState,
+    isMobile,
+    onAdd,
+    onDelete,
+    onEdit,
+    onResetFormState,
+    p2pAdvertiserPaymentMethods,
+}: TPaymentMethodsListContentProps) => {
+    const [isOpen, setIsOpen] = useState(false);
+    const {
+        delete: deleteAdvertiserPaymentMethod,
+        error: deleteError,
+        isSuccess: isDeleteSuccessful,
+    } = p2p.advertiserPaymentMethods.useDelete();
+
+    const { actionType, selectedPaymentMethod } = formState || {};
+    const groupedPaymentMethods = useMemo(() => {
+        const groups: TPaymentMethodsGroup = {};
+        p2pAdvertiserPaymentMethods?.forEach(advertiserPaymentMethod => {
+            if (groups[advertiserPaymentMethod.type]) {
+                groups[advertiserPaymentMethod.type]?.paymentMethods?.push(advertiserPaymentMethod);
+            } else {
+                groups[advertiserPaymentMethod.type] = {
+                    paymentMethods: [advertiserPaymentMethod],
+                    title: PAYMENT_METHOD_CATEGORIES[advertiserPaymentMethod.type],
+                };
+            }
+        });
+        return groups;
+    }, [p2pAdvertiserPaymentMethods]);
+
+    useEffect(() => {
+        if (deleteError) {
+            setIsOpen(true);
+        }
+    }, [isDeleteSuccessful, deleteError]);
+
+    return (
+        <div className='p2p-v2-payment-methods-list'>
+            {isMobile ? null : <AddNewButton isMobile={isMobile} onAdd={onAdd} />}
```

Comment:
can use the isDesktop value from useDevice, and use && instead
```suggestion
            {isDesktop && <AddNewButton isMobile={isMobile} onAdd={onAdd} />}
```